airbyte-integrations/connectors/source-visma-economic/source_visma_economic/spec.yaml

```diff
@@ -0,0 +1,17 @@
+documentationUrl: https://apis.e-conomic.com/
```

Comment:
This is the Airbyte's connector page: `https://docs.airbyte.com/integrations/sources/visma-economic`